tmuxp/session.py

```diff
@@ -124,7 +124,8 @@ def new_window(self,
                    window_name=None,
                    start_directory=None,
                    attach=True,
-                   window_index=''):
+                   window_index='',
+                   window_command=None):
         """Return :class:`Window` from ``$ tmux new-window``.
```

Comment:
Can you add the docstrings (``:param window_index: description``, so on) for these params?